src/Npgsql/NpgsqlConnector.Auth.cs

```diff
@@ -366,8 +366,20 @@ public override long Position
 
         class AuthenticationCompleteException : Exception { }
 
-        string? GetPassword()
+        string? GetPassword(string? username)
         {
+            if (GetDynamicPassword != null){
```

Comment:
Move the curly bracket to the next line.

Reply:
i had a couple of these, must be because im using vscode and there is no omnisharp config. Be nice when they update the csharp plugin to use editorconfig